github_copilot_seats/README.md

```diff
@@ -11,10 +11,18 @@ This automation searches daily for all users in a GitHub organization that are a
 If Copilot was not used in a preceding period, it automatically unregisters them, and then notifies them.
 Users can then optionally ask for their subscription to be reinstated.
 
+## Before Deploying This AutoKitteh Project:
+- Set the `GITHUB_ORG` in the project's vars.
```

Comment:
I think we can determine the org automatically (where the GitHub app was installed) - but that's for a later PR.

Reply:
Do you want me to open a ticket?

Reply:
No - see comment below about probably not needing orgs at all.

Reply:
Whichever you prefer - either just add it now, or open a ticket as a reminder to do this sometime soon. Either way it should be quick.